### PR TITLE
Fixed bug where PR was closed prematurely 

### PR DESCRIPTION
**Pull request by MarconZet**

### CODE REVIEW

File: setup-bsp.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-cs launch -r m2Local org.jetbrains.bsp:bazel-bsp:2.5.1 -M org.jetbrains.bsp.bazel.install.Install -- --bazel-workspace . --directory ../bazel-steward-bsp
+cs launch -r m2Local org.jetbrains.bsp:bazel-bsp:2.5.1 -M org.jetbrains.bsp.bazel.install.Install
```
Comment (MarconZet) [inline setup-bsp.sh:2]: Seems wrong to me

Comment (lukaszwawrzyk) [reply setup-bsp.sh:2]: Try opening ../bazel-steward-bsp instead of bazel-steward. It should generally work better like this.

**Review by lukaszwawrzyk — APPROVED** [review]

File: app/src/main/kotlin/org/virtuslab/bazelsteward/app/App.kt [hunk 2]
```diff
@@ -78,7 +78,7 @@ data class App(
               val openPr = if (prStatus == NONE) {
                 val oldPrs = gitHostClient.getOpenPrs().filter {
                   it.branch.name.startsWith(pr.branchPrefix) && it.branch != pr.branch
-                }
+                }.filter { gitHostClient.checkPrStatus(it.branch) != OPEN_MODIFIED }
```
Comment (lukaszwawrzyk) [inline app/src/main/kotlin/org/virtuslab/bazelsteward/app/App.kt:81]: Ok, this is to not touch a PR that was modified by someone even if we will add a PR for newer version, right?

**Review by lukaszwawrzyk — APPROVED** [review]

File: app/src/main/kotlin/org/virtuslab/bazelsteward/app/App.kt (hunk 2 above)
Comment (MarconZet) [reply app/src/main/kotlin/org/virtuslab/bazelsteward/app/App.kt:81]: Yes

**Pull request merged**